en/complete_vehicles/px4_vision_kit.md

```diff
@@ -0,0 +1,295 @@
+
+In summary, you just insert the provided USB stick into the *UP Core* USB1 port, and then power the vehicle using a 4S battery.
+The USB stick contains a bootable *Ubuntu Linux 18.04* image with PX4 Avoidance software included, which automatically runs the *local planner* for object avoidance and collision prevention on startup.
+
+The avoidance system will be up and running after about 2.5 minutes.
+
+Once started the companion computer can be used both as a computer vision development environment and for running the software.
+
+
+### Login to the Companion Computer {#login_mission_computer}
+
+To login to the companion computer:
+1. Connect a keyboard and mouse to the *UP Core* via its USB2 port:
+
+   ![UP Core: USB2](../../assets/hardware/px4_vision_devkit/upcore_port_usb2.png)
+   - Use the USB-JST cable from the kit to expose a USB A connector
+   
+     ![USB to JST cable](../../assets/hardware/px4_vision_devkit/usb_jst_cable.jpg)
+   - A USB hub can be attached to the cable if the keyboard and mouse have separate connectors. 
+1. Connect a monitor to the *UP Core* HDMI port.
+
+   ![UP Core: HDMI port](../../assets/hardware/px4_vision_devkit/upcore_port_hdmi.png)
+
+   The Ubuntu login screen should then appear on the monitor. 
+1. Login to the *UP Core* using the credentials:
+   - **Username:** px4vision
+   - **Password:** px4vision
+
+### Installing the image on the Companion Computer {#install_image_mission_computer}
+
+You can alternatively install the image on the *UP Core* and boot from internal memory (instead of the USB).
+
+To flash the USB image to the *UP Core*:
+
+1. Insert the pre-flashed USB drive into the *UP Core* USB1 port.
+1. [Login to the companion computer](#login_mission_computer) (as described above).
+1. Open a terminal and run the following command to copy the image onto internal memory (eMMC).
+   The terminal will prompt for a number of responses during the flashing process.
+   ```sh
+   cd ~/catkin_ws/src/px4vision_ros
+   sudo ./flash_emmc.sh
+   ```
+
+   > **Note** All information saved in the *UP Core* computer will be removed when executing this script
+
+1. Pull out the USB stick.
+1. Restart the vehicle.
+   The *UP Core* computer will now boot from internal memory (eMMC).
+
+
+### Developing/Extending PX4 Avoidance
+
+The PX4 Vision’s *UP Core* computer provides a complete and fully configured environment for extending PX4 Avoidance software (and more generally, for developing new computer vision algorithms using ROS2).
+You should develop and test your software on the vehicle, sync it to your own git repository, and share any fixes and improvements with the wider PX4 community on the github [PX4/Avoidance](https://github.com/PX4/avoidance) repo.
+
+The catkin workspace is at `~/catkin_ws`, and is preconfigured for running the PX4 avoidance local planner. 
+The launch-from-boot file (`avoidance.launch`) is in the `px4vision_ros` package
```

Comment:
@Jaeyoung-Lim I moved this stuff out of the "installing the image" because it is true for image on card or on internal memory. 

For launch from boot stuff:
1. YOu say `The launch-from-boot file (`avoidance.launch`) is in the `px4vision_ros` package`. How does the system get the right version of this file? i.e. if I modify it what do I have to do to make it live?
2. Following on from that, see next few lines where we stop the "start on boot" stuff. I assume the model is I stop it (temporarily), change the file `avoidance.launch`, then reboot to restart? If I permanently stop it, how do I restart the service?

Reply:
@Jaeyoung-Lim ^^^^

Reply:
@hamishwillee 1. There is a systemd service which starts the roslaunch file. If you modify the file, the system will use that file automatically. 2. You can actually not stop the service and modify it and either restart the service or reboot. You can start the service using `systemctl start avoidance.service` or permanently enable the service to run on boot using `systemctl enable avoidance.service`

Reply:
Thanks @Jaeyoung-Lim - I'm going to merge, but can you sanity check the changes: https://github.com/PX4/px4_user_guide/pull/653/commits/0b37d741dfb3763e2f09be780b881710dda940b2

Essentially it says that you modify the `avoidance.launch` to specify what planner is launched.